content/en/blog/2021-08-15-seccomp-default.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: blog
+title: "Hardening Kubernetes security by enabling seccomp by default"
+date: 2021-08-15
+slug: seccomp-default
+---
+
+**Author:** Sascha Grunert, Red Hat
+
+_Editor's note: Sascha is member of [SIG Release][sigr] and is working on many
+other different container related topics. Feel free to reach him out on Twitter
+[@saschagrunert][twitter] or via the [Kubernetes slack @sascha][slack]._
+
+[sigr]: https://github.com/kubernetes/sig-release
+[twitter]: https://twitter.com/saschagrunert
+[slack]: https://kubernetes.slack.com/team/U53SUDBD4
+
+---
+
+Applying seccomp profiles to containerized workloads is one of the key tasks
+when it comes to enhancing the security of the application deployment.
+Developers, site reliability engineers and infrastructure administrators have to
+work hand in hand to create, distribute and maintain the profiles over the
+applications life-cycle.
+
+Kubernetes introduced dedicated [API fields][api] in the `SecurityContext` with
+the [graduation of seccomp to General Availability (GA)][ga] in v1.19.0. This
+enhancement allowed an easier way to specify if the whole pod or a specific
+container should run as:
+
+[api]: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#seccompprofile-v1-core
```

Comment:
I'd link to https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1

Reply:
Done :+1: